x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecurityRcs1AliasIT.java

```diff
@@ -275,11 +276,11 @@ public void testAliasUnderRcs1WithMixedPrivilegesWithRealFilterV1() throws Excep
               "indices": [
                 {
                   "names": ["index-alias"],
-                  "privileges": ["read"]
```

Comment:
This file is net-new. The fact that it shows as a diff in this commit is because I didn't add it in the same commit as the "prod-level" changes.